keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/MultivariateGaussianVertex.java

```diff
@@ -122,16 +122,23 @@ public DoubleTensor sampleWithShape(long[] shape, KeanuRandom random) {
     }
 
     private static long[] checkValidMultivariateShape(long[] muShape, long[] covarianceShape) {
```

Comment:
Can we add something to the release notes about this? (especially important given it's a non backwards compatible change.

Reply:
👍 It really should be a vector. Very strange that it had to be a matrix before.